api/filesystem/filesystem.go

```diff
@@ -627,6 +629,25 @@ func (service *Service) CopySSLCertPair(certPath, keyPath string) (string, strin
 	return defCertPath, defKeyPath, nil
 }
 
+// GetDefaultSSLCacertsPath returns the ssl cacert path
+func (service *Service) GetDefaultSSLCacertsPath() string {
```

Comment:
if this is only used internally, then change to private
```suggestion
func (service *Service) getDefaultSSLCACertsPath() string {
```